js/app.js

```diff
@@ -36,12 +36,13 @@ function xor(a, b) {
 }
  */
 // 4
-    function _if(bool, func1, func2) {
-        if (bool) {
-            return func1();
-        }
-        return func2();
+
+function condition(bool, func1, func2) {
```

Comment:
bool is not a good argument name, it only indicates the type of the argument, not the purpose 🥸

The first argument's purpose is to decide if you should call a first function or a second. Maybe chat gpt will come up with a good argument name here 🤔